content/advanced/hooks/example-in-python/index.md

```diff
@@ -25,14 +25,12 @@ Your first step is to use the [CloudFormation Command Line Interface (CLI)](http
 
 :::code{language=shell showLineNumbers=false showCopyAction=true}
 mkdir example-hook
-cd example-hook
-:::
+cd example-hook:::
```

Comment:
```suggestion
cd example-hook
:::
```

Reply:
The suggested formatting breaks the generated code block.

Reply:
Running preview_build helped me fix this